internal/api/ui/login/renderer.go

```diff
@@ -437,8 +437,7 @@ func (l *Login) getTranslator(ctx context.Context, authReq *domain.AuthRequest)
 	if err != nil {
 		logging.OnError(err).Warn("cannot load instance restrictions to retrieve allowed languages for creating the translator")
 	}
-	translator, err := l.renderer.NewTranslator(ctx, restrictions.AllowedLanguages)
-	logging.OnError(err).Warn("cannot load translator")
+	translator := l.renderer.NewTranslator(ctx, restrictions.AllowedLanguages)
```

Comment:
Now that the NewTranslator no longer returns an error, and it uses `MustAddMessages`, that means it can panic. That's ok for the start-up calls of NewTranslator, but I'm not sure if that's desirable in the login renderer. (Can panic during the request).

Even if we call `recover()`, the behavior will change from logging the error and let the request continue, to terminating the request.

Can't we initialize the translator when instantiating the login, like we do for the APIs?